homeassistant/components/sensor/switcher_kis.py

```diff
@@ -0,0 +1,264 @@
+"""Home Assistant Switcher Component.
```

Comment:
there's a new component/platform format. Instead of `component/platform` we use `platform/component` so this and the other files needs to be `switcher_kis/sensor.py`

Reply:
Thank you.
Fixed in [this](https://github.com/home-assistant/home-assistant/pull/20830/commits/6d478ac3f47207b344a670fc3a04f1811c544a83) commit.